Cargo.toml

```diff
@@ -14,9 +14,16 @@ exclude = [
     ".gitignore"
 ]
 
-[dependencies]
-rand = "0.3"
-arrayref = "0.3.2"
+[dependencies.rand]
+optional = true
+version = "0.3"
+
+[dependencies.arrayref]
+version = "0.3.2"
```

Comment:
I went ahead and put this back for now, however the released crate still depends on `std`. I can build my project by pulling in `arrayref` via git, but FYI it'd be nice to bump this whenever there's another release.